src/lib/Components/Inbox/Conversations/SendConversationMessage.ts

```diff
@@ -1,20 +1,29 @@
 import { commitMutation, Environment, graphql, MutationConfig, RecordSourceSelectorProxy } from "react-relay"
 import { ConnectionHandler } from "relay-runtime"
-import RelayProps from "./relayInterfaces"
+
+interface Conversation {
```

Comment:
This mutation needs the following pieces. `last_message_id` is newly added to the schema